CMakeLists.txt

```diff
@@ -127,6 +127,31 @@ else ()
             gazebo_ros
             )
 
+    # Moveit Packages
+    list(APPEND MROVER_PACKAGES
+            moveit_commander
```

Comment:
Wrong place to put this stuff in... scroll up the file and put inside mrover packages